packages/bsky/src/lexicon/lexicons.ts

```diff
@@ -1860,16 +1860,6 @@ export const schemaDict = {
             cursor: {
               type: 'string',
             },
-            rkeyStart: {
-              type: 'string',
-              description:
-                'DEPRECATED: The lowest sort-ordered rkey to start from (exclusive)',
-            },
-            rkeyEnd: {
-              type: 'string',
-              description:
-                'DEPRECATED: The highest sort-ordered rkey to stop at (exclusive)',
-            },
```

Comment:
Were these removed from `lexicons/` in a different PR that didn't include codegen?

Reply:
oh yeah looks like it - I didn't check the codegen close enough: https://github.com/bluesky-social/atproto/pull/2506